go.mod

```diff
@@ -8,16 +8,16 @@ require (
 	github.com/drone/envsubst v1.0.3
 	github.com/go-logr/logr v1.4.2
 	github.com/google/go-cmp v0.6.0
-	github.com/jaypipes/ghw v0.12.0
-	github.com/k8stopologyawareschedwg/deployer v0.21.3
+	github.com/jaypipes/ghw v0.16.0
+	github.com/k8stopologyawareschedwg/deployer v0.22.0
 	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.1.2
 	github.com/k8stopologyawareschedwg/podfingerprint v0.2.2
-	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.20.3
+	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.21.0
 	github.com/kubevirt/device-plugin-manager v1.19.4
 	github.com/mdomke/git-semver v1.0.0
-	github.com/onsi/ginkgo/v2 v2.21.0
-	github.com/onsi/gomega v1.35.1
-	github.com/openshift/api v0.0.0-20240524162738-d899f8877d22
+	github.com/onsi/ginkgo/v2 v2.22.0
+	github.com/onsi/gomega v1.36.1
+	github.com/openshift/api v0.0.0-20250305013520-e7f23be12279
```

Comment:
which version is this? please set an explicit pin (to the tip of the 4.18 branch)

Reply:
this is 4.18 branch, https://github.com/openshift/api/commit/e7f23be12279

> please set an explicit pin (to the tip of the 4.18 branch)
not sure I get that, do you mean appending a comment with the branch like below:
`github.com/openshift/api v0.0.0-20250305013520-e7f23be12279 // release-4.18`